opm/grid/MinpvProcessor.hpp

```diff
@@ -132,11 +147,16 @@ namespace Opm
         if (!actnum.empty() && actnum.size() != log_size) {
             OPM_THROW(std::runtime_error, "Wrong size of ACTNUM input, must have one element per logical cartesian cell.");
         }
+        if (pinchOption4ALL && permz.empty()) {
+            OPM_THROW(std::runtime_error,
+                      "If option 4 of PINCH keyword is ALL, then the deck needs to specify PERMZ or PERMX");
+            }
 
         // Main loop.
         for (int jj = 0; jj < dims_[1]; ++jj) {
             for (int ii = 0; ii < dims_[0]; ++ii) {
                 for (int kk = 0; kk < dims_[2]; ++kk) {
+                    bool option4ALLSupported = false;
                     const int c = ii + dims_[0] * (jj + dims_[1] * kk);
```

Comment:
Could you add a commenting line here about the purpose of this boolean. It is not clear from the name. I had to go down to line 315 to understand it. That would also make it easier to understand the code.